storage/index.js

```diff
@@ -1 +1,15 @@
 // JavaScript source code
+var storageModule;
+
+module.exports() = {
+    init = function (config) {
+        //Use following type of structure to require different types of DB
+        if(config.db.type == 'mongo')
+            storageModule = require('./mongo/index.js');
```

Comment:
You can require it as require('mongo') no need of index.js and ./ as it is in the current path itself.